### DISCUSSION TITLE
Termux on pc installation help

### DISCUSSION THREAD

**Discussion by Mando8000** [termux/termux-app#2679] [Q&A] [answered]
Hello everyone, I am looking for some help here. Last year I had termux going with youtube-dl working great. I could have sworn that I installed it on my surface pro 7 directly from here or maybe some other website without any emulator (I could be wrong). Anyways I can't figure out how to install can someone please be kind enough to give me some instructions. I am using a surface pro 8 windows 11 x64 system.

**Comment by ghost** [comment, marked as answer]
You can try [Windows Subsystem for Android](https://www.thewindowsclub.com/install-windows-subsystem-for-android) and install Termux as if it's an Windows app. works great! or you can install [Termux in a docker container](https://github.com/termux/termux-docker) which only contains basic files to run Termux than having a copy of android emulator, only docker is required

Although you can use Windows Subsystem for Linux, Virtualbox or cygwin/msys2 instead than using Termux on Windows. unless you have a specific reason why

**Reply by Mando8000** [reply]
> You can try [Windows Subsystem for Android](https://www.thewindowsclub.com/install-windows-subsystem-for-android) and install Termux as if it's an Windows app. works great! or you can install [Termux in a docker container](https://github.com/termux/termux-docker) which only contains basic files to run Termux than having a copy of android emulator, only docker is required
> 
> Although you can use Windows Subsystem for Linux, Virtualbox or cygwin/msys2 instead than using Termux on Windows. unless you have a specific reason why

Thank you wmcbtech. I think I need to make some thing's clear. First of all. I barely know what I'm doing lol so please explain things to me as if you were talking to a 5 year old.  The only thing I'm trying to do is download some youtube videos with youtube-dl. I used youtube-dl last year through termux. There was plenty of tutorials on how to do it i'm not sure why I can't find any anywhere anymore. The apks below am I not able to simply download them and install somehow?

![Screenshot d](https://user-images.githubusercontent.com/102404249/160229452-2bdb04f3-25f3-4afe-8f0a-0912c5fa24db.png)

**Reply by ghost** [reply]
> The only thing I'm trying to do is download some youtube videos with youtube-dl. I used youtube-dl last year through termux.

There is a [Win32 executable for `youtube-dl`](https://github.com/ytdl-org/youtube-dl/releases/download/2021.12.17/youtube-dl.exe) or install `youtube-dl` via `pip` if you have python installed. you'd also need to install [`ffmpeg` for windows](https://www.thewindowsclub.com/how-to-install-ffmpeg-on-windows-10) if you need to use additional features in youtube-dl (e.g. extracting audio)

https://github.com/ytdl-org/youtube-dl#installation

> There was plenty of tutorials on how to do it i'm not sure why I can't find any anywhere anymore. The apks below am I not able to simply download them and install somehow?

You must install an Android emulator first. i mentioned above is Microsoft's Android emulator which is compatible for your device or other emulators like Android Virtual Device or android-x86 in virtualbox. but in your case you're just trying to use `youtube-dl` so you may not need these
